cluster/k8s-1.10.4/README.md

```diff
@@ -11,6 +11,7 @@ local machine and are the pushed to a registry which is exposed at
 ```bash
 export KUBEVIRT_PROVIDER=k8s-1.10.4
 export KUBEVIRT_NUM_NODES=2 # master + one nodes
+#export KUBEVIRT_MEMORY_SIZE=8192M # default memory size is 4096M, uncomment this variable only if node needs more ram
```

Comment:
I think it would be nice if information about ```KUBEVIRT_MEMORY_SIZE``` with a short description would appear only in one place, like ```KUBEVIRT_NUM_NODES``` [link](https://github.com/kubevirt/kubevirt/blob/master/docs/getting-started.md#user-content-compile-and-run-it), maybe it's worth to extend the same section. 

The reason why I think in that way...  this env is not always required to modify, let's imagine that I have to add another optional env then should I add it everywhere as well? However, If a specific provider requires a bigger memory size then it should be mentioned in a provider doc.